geopolars/src/geoseries.rs

```diff
@@ -30,8 +30,52 @@ pub enum TransformOrigin {
     Point(Point),
 }
 
+pub enum BroadcastableParameter<'a, T> {
+    Val(T),
+    Series(&'a Series),
+}
+
+impl<'a> From<f64> for BroadcastableParameter<'a, f64> {
+    fn from(val: f64) -> BroadcastableParameter<'a, f64> {
+        BroadcastableParameter::Val(val)
+    }
+}
+
+impl<'a> From<i64> for BroadcastableParameter<'a, i64> {
+    fn from(val: i64) -> BroadcastableParameter<'a, i64> {
+        BroadcastableParameter::Val(val)
+    }
+}
+
+impl<'a> From<u64> for BroadcastableParameter<'a, u64> {
```

Comment:
I guess in the future if we had lots of different types of scalar parameters we could make a macro here.

Reply:
Yeah for sure. I tried to implement this using trait bounds, for example with the num crates ToPrimative (https://docs.rs/num/latest/num/trait.ToPrimitive.html)  you can do something like: 
 
```rust
impl<'a> From<T> for BroadcastParameter<'a,T>
  where T : impl ToPrimative
{
...
}

impl<'a> From<Series> for BroadcastParameter<'a,Series>
{
...
}
```

The problem is that the compiler disallows this because we dont own the Series struct and the owner of that Struct might in a future release implement the trait ToPrimative for Series, breaking the code above. 

So we can either be explicit or create a macro.
 but there is an issue with then also implementing it for Series as it's from a foreign crate